runelite-api/src/main/java/net/runelite/api/Experience.java

```diff
@@ -42,6 +42,11 @@
 	public static final int MAX_VIRT_LEVEL = 126;
 	public static final int MAX_SKILL_XP = 200_000_000;
 
+	/**
+	 * The maximum possible combat level.
+	 */
+	public static final int MAX_COMBAT_LEVEL = 126;
```

Comment:
There is already const named MAX_VIRT_LEVEL in this file

Reply:
MAX_VIRT_LEVEL is for every trainable skill. If a new combat skill was released, MAX_COMBAT_LEVEL would go probably go up. Should I still remove it?